model/Image.php

```diff
@@ -70,7 +70,7 @@ class Image extends File implements Flushable {
 	 * @config
 	 * @var bool Force all images to resample in all cases
 	 */
-	private static $force_resample = false;
+	private static $force_resample = true;
```

Comment:
Can we make this default to true now, particularly now that we're only serving up the resampled version if it is smaller than the original? @hafriedlander suggested it go in to 3.2 at https://github.com/silverstripe/silverstripe-framework/pull/3265#issuecomment-48076794